core/types/receipt_test.go

```diff
@@ -229,6 +229,7 @@ var (
 			PostState:         common.Hash{3}.Bytes(),
 			CumulativeGasUsed: 6,
 			Logs:              []*Log{},
+			Bloom:             Bloom{},
```

Comment:
why are you adding these? doesn't make any difference, afaict

Reply:
Yep, I think it's an artifact from a previous version